studio/pages/new/[slug].tsx

```diff
@@ -162,12 +230,13 @@ export const Wizard = () => {
     setNewProjectLoading(true)
     const data = {
       cloud_provider: PROVIDERS.AWS.id, // hardcoded for DB instances to be under AWS
-      org_id: currentOrg.id,
+      org_id: _pageState.currentOrg?.id,
       name: projectName,
       db_pass: dbPass,
       db_region: dbRegion,
+      db_pricing_plan: dbPricingPlan,
     }
-    const response = await post(`${API_URL}/projects/new`, data)
+    const response = await post(`${API_URL}/projects`, data)
```

Comment:
i'm presuming there's also a corresponding infra/api change?

Reply:
oh this one right https://github.com/supabase/infrastructure/pull/3230

Reply:
yes, need to run the api from this PR supabase/infrastructure#3229